src/core/fonts.js

```diff
@@ -3335,14 +3335,14 @@ var Type1Font = (function Type1FontClosure() {
       cff.globalSubrIndex = new CFFIndex();
 
       var count = glyphs.length;
-      var charsetArray = [0];
+      var charsetArray = ['.notdef'];
       var i, ii;
       for (i = 0; i < count; i++) {
         var index = CFFStandardStrings.indexOf(charstrings[i].glyphName);
```

Comment:
Nit: Would it (any) sense to define `let glyphName = charstrings[i].glyphName;` once at the top of the loop, rather than re-getting it potentially three times per iteration?